homeassistant/components/tellduslive/__init__.py

```diff
@@ -81,6 +82,22 @@
     return True
 
 
+async def async_add_hubs(hass, client, entry_id):
+    """Add the hubs associated with the current client to device_registry."""
+    dev_reg = await hass.helpers.device_registry.async_get_registry()
+    for hub in await client.async_get_hubs():
+        _LOGGER.debug("Connected hub %s", hub['name'])
+        dev_reg.async_get_or_create(
+            config_entry_id=entry_id,
+            connections={('IP', hub['ip'])},
```

Comment:
Is ip allowed as device registry connection type? Do we have a precedence?

Reply:
No, there is not much more to use..

Reply:
Check with @balloob about this. It might be that we think ip is too unstable to represent a connection.

Reply:
We can't get the mac address?

Reply:
Not when connected via the cloud. Then we only get this:

```json
{
    "client": [
        {
            "id": "247140",
            "uuid": "1b936764-7659-4344-91f2-7c0d45e24149",
            "name": "Mj\u00f6vik",
            "online": "1",
            "editable": 1,
            "extensions": 3,
            "version": "1.0.12-52-gf0aaf75",
            "type": "TellStick ZNet Lite v2",
            "ip": "31.208.248.64"
        }
    ]
}
```

The API call is not available at all when using Local, but I have an idea on how to mock it to get: `TellstickZnetLiteV2:ACCA5401B5EA:NEGB6EGQCF:1.0.12-52-gf0aaf75` (`name:mac:secret:version`)